packages/vx-point/src/Point.ts

```diff
@@ -1,4 +1,7 @@
 export default class Point {
+  public x: number = 0;
```

Comment:
@hshoff I'm not sure if you intended these to be `public` or `private`

Reply:
I think they need to be `public` because much of the codebase references them as `point.x/y` 👍